Cu 868651yuz home scaffolding

brownKasey opened the pull request.

brownKasey committed:
#868651yuz[in development] installed tailwind and set up tailwind config

brownKasey committed:
#868651yuz[in development] added Very basic scaffholding for Home Page

MrTofuuu commented:
Task linked: [CU-868651yuz Home Scaffolding](https://app.clickup.com/t/868651yuz)

MrTofuuu committed:
Delete client/package-lock.json

Since we're using PNPM instead of NPM we shouldn't have a `package-lock.json` file

MrTofuuu reviewed (commented):
Use `PNPM` instead of `NPM`

MrTofuuu reviewed (approved):
LGTM
